### DISCUSSION TITLE
Problem with downloading  1.10.5-preo1.exe

### DISCUSSION THREAD

**Discussion by Sloanj44** [freeplane/freeplane#652] [Help]
I downloaded the latest version of Freeplane as well as Java.  For some reason I get the error message below at the end of my install of Freeplane.
I rebooted after installing Java  as well as Freeplane.  I also do not know how to address the Java Runtime Environment 11-17 problem that seems to be stopping Freeplane.
Can someone kindly help.

**Comment by Sloanj44** [comment]
![image](https://user-images.githubusercontent.com/112812812/189507873-79423c9a-12be-4fc3-8d6b-77345f89af48.png)

This is the message I get when I try to finish the download from Freeplane.

![image](https://user-images.githubusercontent.com/112812812/189507900-166f07d1-ab9e-41c7-ad79-9ce039d768f1.png)

**Reply by binkyxz3** [reply]
I have the exact same problem. Was sent to Java did a download. Freeplane does not run. 
I have no interest in dabbling in code for hours to fix such. What in Freeplane doing ???

**Reply by Sloanj44** [reply]
I just went over to the Issues tab and others have also had our problem.  I haven't had a chance to try their solutions yet because I wanted to get to you and give you a heads up.
Good luck on this...
I'll keep trying too.

**Comment by macmarrum** [comment]
Please have a look at the solution provided in the issue
- https://github.com/freeplane/freeplane/issues/649

**Reply by binkyxz3** [reply]
Thanks. I have reverted back to Freeplane-Setup-with-Java-10.14,exe  as per Euro2021
What a waste of time.

**Reply by Sloanj44** [reply]
Thank you, Macmarrum and binkyxz.  I too went back to the earlier version as well and am now up and running.

**Reply by dpolivaev** [reply]
Preview versions also contain setups with Java. You just need to take it.
I do not delete the setup without Java because it seems to be the most popular one.
The differences between the setup packages are explained at https://docs.freeplane.org/getting-started/Distributions.html.

**Reply by autoca** [reply]
Maybe you need a Java version that is NOT from Oracle.
See also https://github.com/freeplane/freeplane/discussions/659
